monkey/infection_monkey/payload/ransomware/ransomware.py

```diff
@@ -3,15 +3,22 @@
 from pathlib import Path
 from typing import Callable, Iterable
 
-from infection_monkey.telemetry.file_encryption_telem import FileEncryptionTelem
-from infection_monkey.telemetry.messengers.i_telemetry_messenger import ITelemetryMessenger
+from common.agent_events import FileEncryptedEvent
+from common.event_queue import IAgentEventQueue
+from common.tags import T1485_ATTACK_TECHNIQUE_TAG, T1486_ATTACK_TECHNIQUE_TAG
+from infection_monkey.utils.ids import get_agent_id
 from infection_monkey.utils.threading import interruptible_function, interruptible_iter
 
 from .consts import README_FILE_NAME, README_SRC
 from .ransomware_options import RansomwareOptions
 
 logger = logging.getLogger(__name__)
 
+RANSOMWARE_PAYLOAD_TAG = "ransomware-payload"
+RANSOMWARE_TAGS = frozenset(
+    {RANSOMWARE_PAYLOAD_TAG, T1485_ATTACK_TECHNIQUE_TAG, T1486_ATTACK_TECHNIQUE_TAG}
```

Comment:
```suggestion
    {RANSOMWARE_PAYLOAD_TAG,  T1486_ATTACK_TECHNIQUE_TAG}
```

I don't think ransomware fits the intent of T1485. Our goal is not to destroy the data or "render stored data irrecoverable".